frontend/packages/console-shared/src/utils/firehose.ts

```diff
@@ -0,0 +1,39 @@
+import { FirehoseResult } from '@console/internal/components/utils';
+import { K8sKind, K8sResourceCommon, K8sResourceKind } from '@console/internal/module/k8s';
+import * as _ from 'lodash';
```

Comment:
Nit: vendor imports should be listed before Console pkg imports.

Reply:
not relevant now